backend/clutch-config.yaml

```diff
@@ -35,7 +35,12 @@ modules:
   - name: clutch.module.envoytriage
   - name: clutch.module.k8s
   - name: clutch.module.kinesis
+  - name: clutch.module.project
+    typed_config:
+      "@type": types.google.com/clutch.config.module.project.v1.Config
+      project_service_override: clutch.service.project
```

Comment:
i've been thinking we could just add component aliasing to the proto a level up, but we can wait on that until the 1.18 refactor where we introduce contexts and use generics :)

so something like 
```yaml
  - name: clutch.module.project
    components:
      - name: clutch.service.project
        substitute: lyft.service.project
    typed_config:
      "@type": types.google.com/clutch.config.module.project.v1.Config
```

Reply:
Oh yeah neat idea